src/NzbDrone.Core/Tv/MovieRepository.cs

```diff
@@ -194,12 +194,13 @@ public PagingSpec<Movie> MoviesWithoutFiles(PagingSpec<Movie> pagingSpec)
 
         public SortBuilder<Movie> GetMoviesWithoutFilesQuery(PagingSpec<Movie> pagingSpec)
         {
-            
             return Query.Where(pagingSpec.FilterExpression)
                              .AndWhere(m => m.MovieFileId == 0)
-                             //this needs to handle the cases from src\nzbdrone.core\tv\moviestatustype.cs
+                             //note: this doesnt respect the availability delay that is used for RssSync
+                             //should be .AndWhere (m=> m.IsAvailable(delay)) where delay corresponds to AvailabilityDelay on Indexers Settings page
```

Comment:
@onedr0p : I wish I could call isAvailable here too.. but wont work here..

also tried to do a more complex series of comparison operations to take into account the delay but i couldnt accomplish that either.. im still looking into this..